menpo/shape/graph.py

```diff
@@ -499,18 +500,21 @@ def find_all_shortest_paths(self, algorithm="auto", unweighted=False):
         algorithm : 'str', see below, optional
             The algorithm to be used. Possible options are:
 
-            ================ =========================================
-            'dijkstra'       Dijkstra's algorithm with Fibonacci heaps
-            'bellman-ford'   Bellman-Ford algorithm
-            'johnson'        Johnson's algorithm
-            'floyd-warshall' Floyd-Warshall algorithm
-            'auto'           Select the best among the above
-            ================ =========================================
+            ====== =========================================
+            'D'    Dijkstra's algorithm with Fibonacci heaps
```

Comment:
Is this true? Was the documentation wrong? Because no other code here seems to suggest the algorithm string has changed?

Reply:
I think it is true. Please check the link manhttps://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csgraph.shortest_path.html . The code also crashes if you pass the previous strings. The code in various tests pass the default value which is auto in both cases. Perhaps was it an older documentation?